src/content/docs/about/getting-started.mdx

```diff
@@ -0,0 +1,110 @@
+
+Once you have completed the installation, you will be ready to start using Daytona to manage and develop your development environments. If you encounter any issues during installation, consult by reaching out to Daytona's [Slack community](https://go.daytona.io/slack) for assistance.
+
+<DocumentList>
+<DocumentListItem
+    title="Install Daytona"
+    subtitle="Learn how to install Daytona on Linux, macOS, or Windows."
+    href="installation/installation"
+    expanded
+  />
+</DocumentList>
+
+## Adding a Git Provider
+
+To interact with Git repositories and manage version control operations, you can add a [Git Provider](/configuration/git-providers) to Daytona. This step is essential for cloning repositories, pushing changes, and managing your codebase effectively.
+
+Daytona supports a wide range of Git Providers, including popular platforms like [GitHub](/configuration/git-providers#github), [GitLab](/configuration/git-providers#gitlab), and [Bitbucket](/configuration/git-providers#bitbucket), as well as self-hosted solutions. Integrating a Git Provider with Daytona allows you to manage your repositories directly within the platform, eliminating the need for repeated authentication.
+
+<br />
+
+<CardGrid gridNumber={3} mobileGridNumber={1} colsMinWidth='150px' colsMinWidthMobile='100px'>
+    <TitleCard icon={githubLogo} title="GitHub" link="/configuration/git-providers#github" imageAlt="'GitHub logo"/>
+    <TitleCard icon={gitlabLogo} title="GitLab" link="/configuration/git-providers#gitlab" imageAlt="GitLab logo"/>
+    <TitleCard icon={bitbucketLogo} title="Bitbucket" link="/configuration/git-providers#bitbucket" imageAlt="BitBucket logo"/>
+</CardGrid>
+
+For more details on available Git Providers and related commands, use the `daytona git-providers` command to explore options and manage your configurations.
+
+<DocumentList>
+<DocumentListItem
+    title="Add a Git Provider"
+    subtitle="Learn how to add a Git Provider to Daytona and start managing your repositories effectively."
+    href="/configuration/git-providers"
+    expanded
+  />
+</DocumentList>
+
+## Installing a Provider
+
+[Provider](/configuration/providers) is a plugin that interfaces with the [Daytona Server](/usage/server), handling development environment deployment and lifecycle management.
+
+Daytona supports a variety of Providers, allowing you to integrate with different platforms based on your needs. Once installed, a Provider will manage the deployment of your development environment directly within Daytona, ensuring integration with your chosen platform.
+
+For more details on available Providers and related commands, use the `daytona provider` command to explore options and manage your configurations.
+
+<DocumentList>
+<DocumentListItem
+    title="Install a Provider"
+    subtitle="Learn how to install a Provider and interface with the Daytona Server."
+    href="/configuration/git-providers"
+    expanded
+  />
+</DocumentList>
```

Comment:
We are missing install and set IDE, as there are some issues you need to do to get VS Code for example to get up and running